File: loiane-training-java/basico/Exer Aula 14_15/Ex24.java
import java.util.Scanner;

public class Ex24 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Digite dois numeros");
        var number1 = scan.nextDouble();
        var number2 = scan.nextDouble();
        double result = 0;

        System.out.println("Digite a operacao que deseja realizar");
        System.out.println("\"+\" - Soma | \"-\" - Subtracao | \"*\" - Multiplicacao | \"/\" - Divisao");
        var operation = scan.next();

        switch (operation) {
            case "+":
                result = number1 + number2;
                break;
            case "-":
                result = number1 - number2;
                break;
            case "*":
                result = number1 * number2;
                break;
            case "/":
                result = number1 / number2;
                break;
            default:
                System.out.println("Operacao incorreta");
        }

        System.out.println("Resultado: " + result);
        if ((int) result % 2 == 0) {
            System.out.println("Resultado: Par");
        } else {
            System.out.println("Resultado: Impar");
        }

        if (result >= 0) {
            System.out.println("Positivo");
        } else {
            System.out.println("Negativo");
        }

        if (result == (int) result) {
            System.out.println("Inteiro");
        } else {
            System.out.println("Decimal");
        }
        scan.close();
    }
}

File: loiane-training-csharp/procedural-poo/c/Ex24.cs
using System;

class Ex24
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite um numero");
        double numero1 = double.Parse(Console.ReadLine());

        Console.WriteLine("Digite outro numero");
        double numero2 = double.Parse(Console.ReadLine());

        Console.WriteLine("Qual operacao deseja realizar?");
        Console.WriteLine("[+] | [-] | [*] - [/]");
        char operacao = char.Parse(Console.ReadLine());

        double result = 0;
        switch (operacao)
        {
            case '+':
                result = numero1 + numero2;
                break;
            case '-':
                result = numero1 - numero2;
                break;
            case '*':
                result = numero1 * numero2;
                break;
            case '/':
                result = numero1 / numero2;
                break;
            default:
                Console.WriteLine("Entrada invalida");
                break;
        }

        Console.WriteLine("Resultado: " + result);
        if (result % 2 == 0)
        {
            Console.WriteLine("Numero e Par");
        }
        else
        {
            Console.WriteLine("Numero e Impar");
        }

        if (result > 0)
        {
            Console.WriteLine("Numero e Positivo");
        }
        else
        {
            Console.WriteLine("Numero e Negativo");
        }

        if (result == Math.Round(result))
        {
            Console.WriteLine("Numero e Inteiro");
        }
        else
        {
            Console.WriteLine("Numero e Decimal");
        }
    }
}